tmt/log.py

```diff
@@ -379,7 +379,7 @@ def filter(self, record: logging.LogRecord) -> bool:
 
 class QuietnessFilter(logging.Filter):
     def filter(self, record: logging.LogRecord) -> bool:
-        if record.levelno not in (logging.DEBUG, logging.INFO):
+        if record.levelno not in (logging.DEBUG, logging.INFO, logging.WARN, logging.WARNING):
```

Comment:
Is there reason why we don't compare on value? Eg. `record.levelno > logging.WARNING` ?

Reply:
Could be shorter, I personally like the explicit listing here. Let's not block on that.